info.yml

```diff
@@ -10,6 +10,11 @@
 
 configuration:
 
+    app_display_mode:
+        type: str
```

Comment:
Hi @RicardoMusch ,
Instead of using a string setting here, I'd prefer to use a boolean to switch the display mode (something like the [modal](https://github.com/shotgunsoftware/tk-multi-publish2/blob/master/info.yml#L121-L126) mode for tk-multi-publish2)
I think it's safer and should avoid your "else" condition.